doc/howtos/building-images.rst

```diff
@@ -78,6 +81,51 @@ the existing user accounts (including root) has a password set, so
 logging into the running system is impossible. Before building an image,
 you must choose a way of interacting with the system after it has booted.
 
+.. NOTE: this section introduces the difference between development and production
+   images first because it is a choice that must be made before building. Choosing
+   architecture, image format and image content are more important than optional
+   build tweaks (sstate, removal of old images), so those come last.
+
+
+Development Images
+------------------
+
+All images provided by the Ostro Project are intended for
+developers and not directly for production use. To avoid having developers
+accidentally build images for real products that have development
+features enabled, you must make explicit changes in ``local.conf`` to
+enable them.
+
+Developers building their own images for personal use can follow these
+instructions to replicate the configuration of the published Ostro OS images. All necessary
+private keys are provided in the ``ostro-os`` repository.
+
+To do this, before building,  edit the :file:`conf/local.conf` configuration file,
+find the line
+with ``# require conf/distro/include/ostro-os-development.inc`` and
+uncomment it. This will also add some recommended software to the ``ostro-image-noswupd``
+reference image, see below for details.
+
+
+Production Images
+-----------------
+
+When building production images, first follow the instructions
+provided in :file:`meta-intel-iot-security/meta-integrity/README.md` for creating your own
```

Comment:
@dbkinder and @gvancuts should we convert that doc to `.rst` and then integrate it in our website autogen documentation?